viewer/src/datamodels/cad/sector/sectorUtilities.ts

```diff
@@ -133,8 +133,13 @@ export function filterCurrentWantedSectors(
     withLatestFrom(wantedObservable),
     flatMap(([loaded, wanted]) => {
       for (const wantedSector of wanted) {
-        if (loaded.metadata.id === wantedSector.metadata.id && loaded.levelOfDetail === wantedSector.levelOfDetail) {
-          return of(loaded);
+        try {
+          if (loaded.metadata.id === wantedSector.metadata.id && loaded.levelOfDetail === wantedSector.levelOfDetail) {
+            return of(loaded);
+          }
+        } catch (error) {
+          // tslint:disable-next-line: no-console
+          console.log(error);
```

Comment:
`consol.error`. Any why are we eating the error?

Reply:
because there shouldn't be an error there, and if there is we don't want the entire pipeline to go down.. This was only due to some errors not being handled and they caused the id undefined error.